appcues/src/main/java/com/appcues/ui/utils/ActivityExt.kt

```diff
@@ -0,0 +1,67 @@
+package com.appcues.ui.utils
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.os.Build.VERSION
+import android.os.Build.VERSION_CODES
+import android.view.View
+import android.view.ViewGroup
+import android.view.inspector.WindowInspector
+import androidx.lifecycle.ViewTreeLifecycleOwner
+import androidx.lifecycle.ViewTreeViewModelStoreOwner
+import androidx.lifecycle.findViewTreeLifecycleOwner
+import androidx.lifecycle.findViewTreeViewModelStoreOwner
+import androidx.savedstate.findViewTreeSavedStateRegistryOwner
+import androidx.savedstate.setViewTreeSavedStateRegistryOwner
+import java.lang.reflect.Method
+
+@Suppress("UNCHECKED_CAST")
+@SuppressLint("PrivateApi")
+internal fun Activity.getParentView(): ViewGroup {
```

Comment:
moved it into a different file since its being used in other places and its generic enough to be extracted